catalyst/runners/runner.py

```diff
@@ -30,6 +31,33 @@
 from catalyst.utils.tracing import save_traced_model, trace_model
 
 
+def resolve_bool_fp16(fp16: Union[Dict, bool]):
```

Comment:
could we make it `_resolve_bool_fp16`? or move to `catalyst/runner/functional.py`? I just want to keep `runner.py` clean and clear :)

Reply:
Done